test/format-test-results.spec.ts

```diff
@@ -0,0 +1,130 @@
+import * as jsonModule from '../src/lib/json';
+
+import { extractDataToSendFromResults } from '../src/cli/commands/test/formatters/format-test-results';
+import { Options } from '../src/lib/types';
+const fs = require('fs');
```

Comment:
Why not use import like the other files?

```suggestion
import * as fs from 'fs';
```

Reply:
done